blockwork/activities/workflow.py

```diff
@@ -23,3 +31,20 @@ def wf() -> None:
     In the future we may want to add common options such as --dryrun here
     """
     pass
+
+
+@click.command(name="_wf_step", hidden=True)
+@click.argument("spec_path", type=click.Path(dir_okay=False, exists=True, path_type=Path))
+@click.pass_obj
+def wf_step(ctx: Context, spec_path: Path):
+    """
+    Loads a serialised transform specification from a provided file path, then
+    resolves the transform class and executes it. This should NOT be called
+    directly but instead as part of a wider workflow.
+    """
+    spec = json.loads(spec_path.read_text(encoding="utf-8"))
+    # Import the relevant transform
+    mod_spec = importlib.import_module(spec["mod"])
+    transform_cls: type[Transform] = getattr(mod_spec, spec["name"])
+    logging.info(f"Running serialised {transform_cls.__name__}")
+    transform_cls._run_serialized(ctx, spec)
```

Comment:
We will need to add cache store functionality (probably here) if we want parallel to work with caching (doesn't have to be this PR).

We'll probably want to make the non-parallel functionality also run wf_step so we aren't duplicating

Reply:
Added a big TODO covering these points